client/my-sites/sharing/connections/services/facebook.js

```diff
@@ -0,0 +1,59 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { SharingService, connectFor } from 'my-sites/sharing/connections/service';
+
+export class Facebook extends SharingService {
+	static propTypes = {
+		...SharingService.propTypes,
+	};
+
+	static defaultProps = {
+		...SharingService.defaultProps,
+	};
+
+	didKeyringConnectionSucceed( availableExternalAccounts ) {
+		if ( availableExternalAccounts.length === 0 ) {
+			this.props.failCreateConnection( {
+				message: [
+					this.props.translate(
+						'The Facebook connection could not be made because this account does not have access to any Pages.',
```

Comment:
:information_source: This string is probably long enough that it doesn't need a **context**. Are you sure you don't want to use a translator comment instead?